content/docs/command-reference/exp/list.md

```diff
@@ -5,8 +5,8 @@ List experiments in a <abbr>DVC repository</abbr> (remote or local).
 ## Synopsis
 
 ```usage
-usage: dvc exp list [-h] [-q | -v] [--rev <rev>]
-                    [--all] [--names-only]
+usage: dvc exp list [-h] [-q | -v] [-A] [--rev <commit>]
+                    [-n <num>] [--names-only]
```

Comment:
not related: `--names-only` description is missing it seems like

Reply:
Added #3195 for this. I can merge one of these and rebase the other.